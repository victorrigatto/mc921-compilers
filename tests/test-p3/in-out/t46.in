int main() {
    int x[2][2] = { {1, 2, 3}, {3, 4, 5} };
    return 0;
}
