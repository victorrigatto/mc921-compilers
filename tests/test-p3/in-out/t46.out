SemanticError: List & variable have different sizes @ 2:9
